src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -123,68 +124,73 @@ class Response
      * @var array
      */
     public static $statusTexts = array(
-        100 => 'Continue',
```

Comment:
please revert that change, we prefer using numbers than consts here

Reply:
Why? :-)

Reply:
Because the real standard is HTTP, not constant names.

Reply:
I think they are more readable with numbers than with consts.  👍 
Changes reverted.